README.md

```diff
@@ -1,45 +1,36 @@
-```
-4.	If on Windows, open “PowerShell” and “cd” to the directory that you created is step 2. Then run the following commands in that directory (only use the text after the command prompts below):
+
+- **Install node (19.0.1 tested) & npm (9.1.1 tested)**
+
+- **Clone repo**
 ```
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing> cd TetheredRing
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing> npm install
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing> npm run dev
+git clone https://github.com/philipswan/TetheredRing.git
 ```
-5.	Open Chrome (note: other browsers may work too but just are untested) and paste the url provided by the "npm run dev" command into the browser (note: this url is probably "http://localhost:3000/").
-6.	You should immediately see a sphere with a ring on it appear. You should be able to interact with it. If you press the "W" key, this should jump you up close to a transit terminus on the ring. However, at this point the earth, and other objects, are missing its texture.
-7. To fix this (on Windows), enter the following commands...
+- **Setup**
 ```
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing> cd textures
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> wget https://www.project-atlantis.com/wp-content/threejs-simulation/textures/bluemarble_16384.png -outfile bluemarble_16384.png
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> curl.exe -O https://www.project-atlantis.com/wp-content/threejs-simulation/textures/moon.jpg
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> curl.exe -O https://www.project-atlantis.com/wp-content/threejs-simulation/textures/movingRingTexture.jpg
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> curl.exe -O https://www.project-atlantis.com/wp-content/threejs-simulation/textures/myakka_oli_2022031_lrg.jpg
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> mkdir 24x12\LR
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> mkdir 24x12\HR
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> cd 24x12\LR
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures\24x12\LR> foreach ($X in @(0..23)) {foreach ($Y in @(0..11)) {curl.exe -O https://www.project-atlantis.com/wp-content/threejs-simulation/textures/24x12/LR/earth_LR_24x12_${X}x${Y}.jpg}}
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures\24x12\LR> cd ..\HR
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures\24x12\HR> foreach ($X in @(0..23)) {foreach ($Y in @(0..11)) {curl.exe -O https://www.project-atlantis.com/wp-content/threejs-simulation/textures/24x12/HR/earth_HR_24x12_${X}x${Y}.jpg}}
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures\24x12\HR> cd ..\..\..
+cd TetheredRing
+npm install
+npm run dev-sync
```

Comment:
```suggestion
Instruct the node package manager to run a script called "syncTextures.js" that will download any missing texture files from the project-atlantis.com website, since these are too large to be stored in the git repo. (Advanced user tip: If you've already downloaded the textures to another local instance of the project, you can save time by manually copying the textures from the other instance, and then run this command just to pick up any missing files. Be warned, however, that the script does not detect if the contents of any files have changed.)
**npm run dev-sync**
```